src/ResourceManager/Network/Commands.Network/help/New-AzureRmVirtualHubRouteTable.md

```diff
@@ -0,0 +1,86 @@
+---
+external help file: Microsoft.Azure.Commands.Network.dll-Help.xml
+Module Name: AzureRM.Network
+online version: https://docs.microsoft.com/en-us/powershell/module/azurerm.network/new-azurermvirtualhubroutetable
+schema: 2.0.0
+---
+
+# New-AzureRmVirtualHubRouteTable
+
+## SYNOPSIS
+Creates an Azure Virtual Hub Route Table object.
+
+## SYNTAX
+
+```
+New-AzureRmVirtualHubRouteTable
+ -Route <System.Collections.Generic.List`1[Microsoft.Azure.Commands.Network.Models.PSVirtualHubRoute]>
```

Comment:
array, not list

Reply:
Already fixed in code - regenerated help.